addons/events/fnc_globalEventJIP.sqf

```diff
@@ -40,4 +55,7 @@ GVAR(eventNamespaceJIP) setVariable [_jipID, [EVENT_PVAR_STR, [_eventName, _para
 // execute on every machine
 [QGVAR(eventJIP), [_eventName, _params]] call CBA_fnc_globalEvent;
 
+// Remove JIP once obj is deleted
+if (!isNil "_obj") then { [_jipID, _obj] call FUNCMAIN(removeGlobalEventJIP); };
```

Comment:
Despite it being technically correct, I wouldn't use the `FUNCMAIN` macro as it's not used anywhere else in CBA.

Reply:
```suggestion
if (!isNil "_obj") then { [_jipID, _obj] call CBA_fnc_removeGlobalEventJIP };
```